tests/dashboard/test_product.py

```diff
@@ -678,3 +679,26 @@ def test_assign_collection_to_product(product_in_stock):
     form.save()
     assert product.collections.first().name == 'test_collections'
     assert collection.products.first().name == product.name
+
+
+def test_sanitize_product_description(product_type, default_category):
+    product = Product.objects.create(
+        name='Test Product', price=10, description='', pk=10,
+        product_type=product_type, category=default_category)
+    data = model_to_dict(product)
+    data['description'] = (
+                          '<b>bold</b><p><i>italic</i></p><h2>Header</h2>'
+                          '<h3>subheader</h3><blockquote>quote</blockquote>'
+                          '<p><a href="www.mirumee.com">link</a></p>'
+                          '<p>an <script>evil()</script>example</p>')
+    data['price'] = 20
+    form = ProductForm(data, instance=product)
+    assert form.is_valid()
+    form.save()
+    assert product.description == (
+                                  '<b>bold</b><p><i>italic</i></p>'
+                                  '<h2>Header</h2><h3>subheader</h3>'
+                                  '<blockquote>quote</blockquote>'
+                                  '<p><a href="www.mirumee.com">link</a></p>'
+                                  '<p>an &lt;script&gt;evil()&lt;/script&gt;'
+                                  'example</p>')
```

Comment:
Is this indentation correct? Same above.

Reply:
Seems like its aligned with opening bracket, something we want to avoid as it turns one line edit to multiple line edit on eventual changes.